app/views/content/train-to-be-a-teacher/accessibility-adjustments.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Adjustments to help you train to teach"
+heading: "Adjustments to help you train to teach"
+subcategory: Postgraduate teacher training
+description: |-
+  Find out about adjustments teacher training providers can make to help you train.
+cta_adviser:
+  adviser:
+    text: Talk to an adviser with years of teaching experience about your different training and funding options. Chat by phone, text or email, as little or as often as you need.
+calls_to_action:
+  find:
+    name: find
+navigation: 20.25
+navigation_title: Adjustments to help you train
+navigation_description: Find out about adjustments teacher training providers can make to help you train.
+cta_arrow_link:
+  financial-support:
+    link_target: "/funding-and-support/if-youre-disabled"
+    link_text: "Find out more about financial support if you're disabled"
+---
+
+Teacher training providers can make adjustments if you need supoprt to become a teacher because you're neurodiverse, have a long-term physical or mental health condition, or have any other accessibility needs. 
+
+The experience and perspective of a diverse workforce is invaluable in schools. These adjustments help make sure that anyone interested in becoming a teacher feels confident and comfortable to be the best teacher they can be.
+
+## Adjustments when you apply 
+
+During the application process, providers could make adjustments such as: 
```

Comment:
```suggestion
During the application process, providers can make adjustments such as: 
```
Should this be 'can' rather than 'could' - do providers get the choice?